ui/app/components/ui/boxed-message/index.scss

```diff
@@ -0,0 +1,18 @@
+.boxed-message {
+  &__container {
+    border: 1px solid;
+    border-radius: 5px;
+    padding: 18px 12px;
+    display: flex;
+    @extend %font;
+    font-size: 12px;
+  }
+
+  &__icon {
```

Comment:
Nit: empty block

Reply:
this was fixed in https://github.com/MetaMask/metamask-extension/pull/8071/commits/2d4b2ed5d84a48f0d1345bb53d29db405f003e5e